tide_site.install

```diff
@@ -347,3 +347,74 @@ function tide_site_update_8018() {
     $field->setSetting('file_extensions', 'png gif jpg jpeg svg')->save();
   }
 }
+
+/**
+ * Add a new optional Twitter Image on the Site Taxonomy.
+ */
+function tide_site_update_8019() {
+
+  $config_location = [Settings::get('config_sync_directory', FALSE)];
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  try {
+    _tide_read_config('field.storage.taxonomy_term.field_site_twitter_image', $config_location, TRUE);
+  }
+  catch (Exception $ex) {
```

Comment:
Catching a general exception makes me a bit twitchy... Does it raise a particular exception that we can capture?